src/firefighter/slack/tasks/reminder_postmortem.py

```diff
@@ -72,3 +75,22 @@ def publish_fixed_next_actions(
         },
         client=client,
     )
+
+
+def send_reminder(incident: Incident, *, to_channel: bool) -> None:
+    prefetched_roles = getattr(incident, "roles_prefetched", [])
+    if to_channel:
+        # Send a message in the incident channel
+        incident.conversation.send_message_and_save(
+            SlackMessageChannelReminderPostMortem(incident)
+        )
+    else:
+        # Send a private message to commander
```

Comment:
Here there is another assumption, that the prefetched_roles only include commander.